packages/admin-test-utils/package.json

```diff
@@ -72,7 +72,7 @@
   },
   "devDependencies": {
     "@reduxjs/toolkit": "1.9.7",
-    "@strapi/pack-up": "workspace:*",
+    "@strapi/pack-up": "4.23.0",
```

Comment:
This comment belongs in strapi/pack-up but since there's nowhere to comment there, I'll put it here.

It's a bit weird that the version matches the current strapi version when it's supposed to be standalone. And considering it's got the @strapi/ prefix I'm worried either a script or someone unaware will attempt to match the version during the release process and cause headaches all around. Do you think we should increase it to something else like 5.0/6.0 or maybe if possible unpublish it on npm and release it as 1.0.0?

Reply:
> It's a bit weird that the version matches the current strapi version when it's supposed to be standalone.

Is it? it's a library, the version on NPM is `4.23.0` so when we publish a new one it'll increment following semver? We're just moving where the code lives from one place to another no? 

You also can't unpublish things because projects depend on it, you can only deprecate it, but i don't think we want to do that either 🤔 the repo still needs to use "a version"

We could release it at v5 i suppose, i'm not sure the changes that are on `v5/main` and how the code changed. maybe that's better – BC to v5 & upgrade `v4` & `v5` with `pack-up@5`?